README.md

```diff
@@ -31,3 +31,19 @@ Used to standardize conventions across external interfaces.
 | HelperHeading       | DesiredHeading, GPS, HelperAISShip |
 | HelperLatLon        | GlobalPath, GPS, HelperAISShip     |
 | HelperSpeed         | GPS, HelperAISShip, WindSensor     |
+
+## Existing Libraries
+
+Possible to-add existing ROS libraries based on ros2 repository:
+[common_interfaces](https://github.com/ros2/common_interfaces) branch ```humble```.
```

Comment:
Only need 1 backtick for inline code. Triple backtick is needed for code blocks

Reply:
TIL. Thanks